### DISCUSSION TITLE
need to navigate cad model with mrtk3 list

### DISCUSSION THREAD

**Discussion by juansuerorobotics** [MixedRealityToolkit/MixedRealityToolkit-Unity#587] [Q&A]
I have a need to navigate cad model with mrtk3 list.

The list should be scrollable, so that i dont have to have a giant list of parts take up the screen.
The list should be resizable, so that the user can decide to show 15 results at a time as opposed to the default 10.
THe list should be hierarchical, kinda like the object hierachy itself since thats the source of the elements of the list.. the objects/parts of the CAD model
I should be able to click on an element of the list corresponding to that part or that subassembly and turn around and be able to light that up in the model or whatever thing that needs to happen to that part of the CAD model.

I saw that you had the databinding framework but i couldnt get my head around it.

I also saw you had a virtualized list but had some trouble using it also.

I also need this to work on a Meta Quest 3 headset as well as Hololen 2 ( maybe even Think Reality A3 )

Finally and i could probably figure this out myself.. i need to be able to filter the list by a string, or some checkboxes

Basically some tree view of the CAD model MRTK style
